### PR TITLE
Create AboutPage component

### PR DESCRIPTION
**Pull request by dylanmullarkey**
Setup the basic outline for the About page. Limited styling as we do not have Figma to go off of yet. Updated React Router to route to page with /about 
In relation to ticket https://github.com/chingu-voyages/v50-tier1-team-03/issues/53

### CODE REVIEW

File: src/components/AboutPage.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Header from './Header'
+
+export default function AboutPage(){
+    return(
+        <div className='about-page-container'>
+            <Header/>
+            <div className='about-page-content'>
+                <img style={{width:"100px"}}src="src\assets\img-placeholder.png" alt="" />
+                <div>
+                    <h1>Griddle and GitHub Grill</h1>
```
Comment (michaella23) [inline src/components/AboutPage.jsx]: I think the `Header` already has an `h1` with the name of the restaurant.

File: src/index.css [hunk 2]
```diff
@@ -40,6 +40,27 @@ body {
...
+/* AboutPage component */
+.about-page-container{
+  width: 70%;
+  margin: 0 auto;
+}
+
+.about-page-content{
+  margin: 0 auto 2em;
+  display: flex;
+  flex-direction: row;
```
Comment (michaella23) [inline src/index.css]: Default `flex-direction` is `row`. Not a big deal, just wanted to mention this line could be omitted.

File: src/components/AboutPage.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import Header from './Header'
+
+export default function AboutPage(){
+    return(
+        <div className='about-page-container'>
+            <Header/>
+            <div className='about-page-content'>
+                <img style={{width:"100px"}}src="src\assets\img-placeholder.png" alt="" />
```
Comment (michaella23) [inline src/components/AboutPage.jsx]: The image is part of the `Header` component if you want to add these attributes there.

**Review by michaella23 — CHANGES_REQUESTED** [review]
Just needs a few quick fixes. thanks @dylanmullarkey!

File: src/components/AboutPage.jsx (hunk 1 above)
Comment (dylanmullarkey) [reply src/components/AboutPage.jsx]: I will remove it, but just going off the wireframe given. I think we are loading the restaurant name in our index.html file, which is wrong. 
image

File: src/index.css (hunk 2 above)
Comment (dylanmullarkey) [reply src/index.css]: Good catch! DRY code is important. Thank you

File: src/components/AboutPage.jsx (hunk 3 above)
Comment (dylanmullarkey) [reply src/components/AboutPage.jsx]: Added

**Comment by dylanmullarkey** [comment]
@michaella23 Fixed issues you responded with, I appreciate your feedback. Once approved I can merge.

**Review by michaella23 — APPROVED** [review]
I still think the logo should be in the Header component, but let's try it as is for now.

**Review by michaella23 — APPROVED** [review]

**Review by michaella23 — APPROVED** [review]

**Review by michaella23 — APPROVED** [review]

**Pull request merged**